left_arm:
  P1:
    cx: 53.0
    cy: 390.0
    ik_x: 0.2
    ik_y: 0.22
  P2:
    cx: 37.0
    cy: 158.0
    ik_x: 0.45
    ik_y: 0.3
  P3:
    cx: 273.0
    cy: 170.0
    ik_x: 0.45
    ik_y: 0.05
  P4:
    cx: 262.0
    cy: 403.0
    ik_x: 0.2
    ik_y: 0.05
right_arm:
  P1:
    cx: 595.0
    cy: 393.0
    ik_x: 0.2
    ik_y: -0.22
  P2:
    cx: 620.0
    cy: 164.0
    ik_x: 0.45
    ik_y: -0.3
  P3:
    cx: 385.0
    cy: 164.0
    ik_x: 0.45
    ik_y: -0.05
  P4:
    cx: 387.0
    cy: 398.0
    ik_x: 0.2
    ik_y: -0.05
